src/main/java/com/conveyal/gtfs/graphql/fetchers/JDBCFetcher.java

```diff
@@ -189,9 +199,6 @@ public static GraphQLFieldDefinition field (String tableName) {
         for (String key : argumentKeys) {
             // Limit and Offset arguments are for pagination. projectStops is used to mutate shape points before
             // returning them. All others become "where X in A, B, C" clauses.
-            Set<String> argsToSkip = new HashSet<>();
```

Comment:
The comment above only mentions pagination and projectStops, saying that all others become "where" clauses. In fact it looks like we're now skipping paginationArgs, boundingBoxArgs, and dateTimeArgs. The comment should be updated.

Reply:
This comment is actually referencing the old file in the diff.  However, the argsToSkip variable in the new file is lacking a good comment, so I will update it.